libs/platform/src/lib/icon-tab-bar/public_api.ts

```diff
@@ -1,2 +1,11 @@
 export * from './icon-tab-bar.module';
 export * from './icon-tab-bar.component';
+export * from './interfaces/icon-tab-bar-item.interface';
+export * from './interfaces/tab-config.interface';
+export * from './directives/dnd/icon-bar-dnd-container.directive';
+export * from './directives/dnd/icon-bar-dnd-item.directive';
+export * from './directives/dnd/icon-bar-dnd-list.directive';
+export * from './directives/extra-button/extra-button.directive';
+export * from './constants';
+export * from './types';
+export * from './components';
```

Comment:
there're a lot of internal stuff here, should we really export all of these?

Reply:
Yes. Otherwise angular will export them by itself, assign "internal name".
Later, the same story will happen in another sub-package, and "internal name" will be the same as in prev. subpackage.
And finally, when root platform package is going to be built, it will fail with an error that "subpackage A already contains exported member called n"